Split `translating.rst` into pages for Translators and Coordinators (#1575)
python/devguide: documentation/translations/coordinating.rst

============
Coordinating
============

Starting a new translation
==========================

First subscribe to the `translation mailing list <translation_ml_>`_,
and introduce yourself and the translation you're starting. Translations
fall under the aegis of the `PSF Translation Workgroup <translation_wg_>`_

Then you can bootstrap your new translation by using `cookiecutter
<https://github.com/JulienPalard/python-docs-cookiecutter>`__ or
`bootstrapper <https://github.com/python-docs-translations/python-docs-bootstrapper>`__.

The important steps look like this:

- Create the GitHub repo (anywhere) with the right hierarchy (using one

>   of the bootstrappers).

Author: I'll be changing this section later as I want to include some other options, e.g. the transifex route.

- Gather people to help you translate. You can't do it alone.
- You can use any tool to translate, as long as you can synchronize with Git.
  Some use Transifex, and some use only GitHub. You can choose another
  way if you like; it's up to you.

> - Ensure we update this page to reflect your work and progress, either via a
>   PR or by asking on the `translation mailing list <translation_ml_>`_.

Author: ```suggestion
- Update :doc:`this page <translating>` to reflect your work and progress, either via a
  PR or by asking on the `translation mailing list <translation_ml_>`_.
```
The table I left on the page for translators.

>   PR or by asking on the `translation mailing list <translation_ml_>`_.

Reviewer: Good catch on the link!

Coordinating FAQ
================

Are there tools to help in managing the repo?
---------------------------------------------

Here's what we're using:

- :pypi:`pomerge` to propagate translations from one file to others.
- :pypi:`pospell` to check for typos in ``.po`` files.
- :pypi:`powrap` to rewrap the ``.po`` files
  before committing. This helps keep Git diffs short.
- :pypi:`potodo` to list what needs to be translated.
- :pypi:`sphinx-lint` to validate reST syntax in translation files.

More related tools and projects can be found in the
`python-docs-translations`__ organisation on GitHub.

__ https://github.com/python-docs-translations

How is a coordinator elected?
-----------------------------

There is no election; each translation has to sort this out.  Here are some suggestions.

-  Coordinator requests are to be public on the `translation mailing list <translation_ml_>`_.
-  If the given language has a native core dev, the core dev has their
   say on the choice.
-  Anyone who wants to become coordinator for their native language and shows
   motivation by translating and building a community will be named
   coordinator.

> -  In case of concurrency between two persons, no one will sort this out
>    for you.  It is up to you two to organize a local election or whatever is
>    needed to sort this out.

Reviewer: ```suggestion
-  We expect the local community to self-organize coordinators and contributors.
    If you have questions, please ask on the mailing list or Discourse.
```

-  If a coordinator becomes inactive or unreachable for a long

>    period of time, someone else can ask for a takeover on the `translation mailing list <translation_ml_>`_.

Reviewer: ```suggestion
   period of time, someone else can ask to be added as a primary coordinator on the `translation mailing list <translation_ml_>`_.
   As a community resource, we aim to keep translations up to date with active contributors, including coordinators.
```
